virttest/virsh.py

```diff
@@ -3189,14 +3279,17 @@ def cpu_stats(name, options, **dargs):
     return command(cmd, **dargs)
 
 
-def change_media(name, device, options, wait_change_event=False, event_timeout=7, **dargs):
+@EventTracker.wait_event
+def change_media(name, device, options, wait_for_event=False,
```

Comment:
change_media is primitive function ,which is the same with others in this file, so we may keep it as it is , and create another such as change_media_xx() as alternative one.

Reply:
I don't think so. The whole point is to make sure the command will return only if the target event shows. If we create another function, there would be tons of new functions to be created, we don't want that, right?

Reply:
@kylazhang 
@dzhengfy 
please provide your inputs

Reply:
https://github.com/autotest/tp-libvirt/pull/3363/files
I think part of  pr is related to this 3363.
Kyla suggested wait_for_event under change-media is limited to specific sets of cases, which clues can be gotten from 3363,
We don't want to apply wait_for_event on cases with using change-media command

Reply:
@chunfuwen After talk to kyla, I think I understand what did you mean, and I'm very glad to explain the question you had:
See this arg 'wait_for_event=False' ? It's a sign of whether we need to wait for event or not, and it's False by default. 
I'm pretty sure you will understand the logic after my explanation.  You're welcome~

Reply:
commands  implementation in virsh.py is primitive. If apply one decorator on change_media, it will go decorator inner when virsh.event is called each time.  It sounds it works from functionality point,but  It is against the style of virsh.py(primitive). 
Considering only few cases  in change media need wait for events, Special cases aren't special enough to break the rules(zen of python)

Reply:
I think there is no need to write new cmd for change_media as in the cases there will be only 1 cmd for change_media if only need once operation, we don't want to add if else in the code and just an parameter with different value should be ok in cfg file. Also checking the wrapper if wait_for_event=False it will return the function itself directly. We can do some test to make sure it can work. And the advantage of decorator is that when other cmd also need this function, it will be easy to update.